### DISCUSSION TITLE
TunedTabPFNRegressor (metric=MetricType.ACCURACY)

### DISCUSSION THREAD

**Discussion by wandimmuufaantaa** [PriorLabs/tabpfn-extensions#43] [Q&A]
MetricType.ACCURACY is being used in both **TunedTabPFNClassifier** and **TunedTabPFNRegressor**. I am wondering how 'accuracy' is being used as a metric for the regression model. 
I tried to use 'mse' and 'rmse' but the model performance is less than the result obtained using 'accuracy'.

```
from tabpfn_extensions.hpo import TunedTabPFNRegressor

# Regression
X, y = load_diabetes(return_X_y=True)
X_train, X_test, y_train, y_test = train_test_split(
    X,
    y,
    test_size=0.33,
    random_state=42,
)
# metric: 'MetricType' = <MetricType.ACCURACY: 'accuracy'>
reg = TunedTabPFNRegressor(n_trials = 100, 
                                     n_validation_size = 0.2, 
                                     metric = 'accuracy',
                                     device = device, 
                                     random_state = 10,
                                     categorical_feature_indices = None, 
                                     verbose = True
                                     )
reg.fit(X_train, y_train)
predictions = reg.predict(X_test)
print("Mean Squared Error (MSE):", mean_squared_error(y_test, predictions))
print("Mean Absolute Error (MAE):", mean_absolute_error(y_test, predictions))
print("R-squared (R^2):", r2_score(y_test, predictions))
